server/frontend/src/components/Notifications/TasksFailed.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="row override-row">
+    <small class="col-md-1">
+      Received {{ notification.timestamp | formatDate }}
+    </small>
+    <span class="label label-danger">Tasks failed</span>
+    <span class="description">
+      {{ notification.description }}
+    </span>
+    <button type="button" class="close" v-on:click="dismiss">
+      <span aria-hidden="true" title="Dismiss">&times;</span>
+    </button>
+    <div class="btn-group pull-right" role="group">
+      <a class="btn btn-default" :href="notification.target_url">View pool</a>
+      <a class="btn btn-info" :href="notification.actor_url">
+        View failed task
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+import { errorParser, formatClientTimestamp } from "../../helpers";
+import * as api from "../../api";
+
+export default {
```

Comment:
This is fine for now but we may want to create a generic class for all notifications and extend them where needed,